login_form.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>Setmore Login</title>
+    </head>
+    <style>
+        input{
+            width: 15%;
+            padding: 10px;
+            margin: 10px 5px;
+            border: 4px solid #ccc;
+            border-radius: 500px;
+            background-color: #bce2c5;
+        }
+        button{
+            padding: 10px 20px;
+            margin: 10px 5px;
+            border: 4px solid #ccc;
+            border-radius: 50px;
+            background-color: #4CAF50;
+            color: white;
+            font-size: 20px;
+        }
+        body{
+            background-color: #2a78c2;
+            font-family: Arial, sans-serif;
+            align-items: flex-end;
+            display: flex;
+            align-items: center;
```

Comment:
Why two align items added ?

Reply:
It was added by mistake. Got to know that previously added properties overrides and worked as expected